### ISSUE TITLE
CSV -> XLSX conversion fails with an error

### ISSUE THREAD

**Issue by niryariv** [hasadna/datacity-k8s#230]
1. Uploaded CSV file, views correctly [here](https://netanya.datacity.org.il/dataset/neighborhoods/resource/f0c71601-2ac5-489b-80b8-dc15564a7b59)
2. Attempting to upload to datastore via admin UI fails with a UnicodeEncodeError message:
![Screen Shot 2021-06-06 at 12 08 51](https://user-images.githubusercontent.com/1771/120918989-f700a580-c6bf-11eb-9285-3d1fde6d799c.png)

**Comment by OriHoch** [comment]
duplicate of #213 explanation in this comment: https://github.com/hasadna/datacity-k8s/issues/213#issuecomment-830843900

**Comment by niryariv** [comment]
@OriHoch Since this happens when we do the conversion (users are instructed to avoid uploading XLS files) and it seems to break on specific characters like `"` (other files with Hebrew headers convert ok) is it possible to add some code that removes these characters from the header row before starting the XLSX conversion? How much work do you estimate?

**Comment by OriHoch** [comment]
I don't see how it is related to XLSX conversion, this is the bug as I understand it:

**reproduction steps**

* upload a csv file which contains special characters (like quotes) in a field name
* check resource view ([example](https://netanya.datacity.org.il/dataset/neighborhoods/resource/f0c71601-2ac5-489b-80b8-dc15564a7b59))
* check datastore upload log ([example](https://netanya.datacity.org.il/dataset/neighborhoods/resource_data/f0c71601-2ac5-489b-80b8-dc15564a7b59))

**expected**

* resource view page: shows green data api button and data dictionary displayed below the table
![image](https://user-images.githubusercontent.com/1198854/120975044-7c945c00-c779-11eb-855a-8198b40ab3a9.png)
![image](https://user-images.githubusercontent.com/1198854/120975103-8a49e180-c779-11eb-8462-dfac52d3fe66.png)
* upload log shows status completed

**actual**

* resource view page: no green data api button, data dictionary is not displayed below the table
![image](https://user-images.githubusercontent.com/1198854/120974910-58d11600-c779-11eb-89c9-6e6396661a9f.png)
![image](https://user-images.githubusercontent.com/1198854/120974939-5ff82400-c779-11eb-8498-bd55cbafed8c.png)
* upload log shows status failed
![image](https://user-images.githubusercontent.com/1198854/120974998-6eded680-c779-11eb-8f94-671fa54f8633.png)

**Comment by OriHoch** [comment]
I estimate 5-10 hours to fix the bug I described in previous comment
